SteamKit2/SteamKit2/Steam/Handlers/SteamAuthentication/SteamAuthentication.cs

```diff
@@ -117,31 +117,33 @@ internal AuthPollResult( CAuthentication_PollAuthSessionStatus_Response response
         /// </summary>
         public class AuthSession
         {
-            internal SteamUnifiedMessages.UnifiedService<IAuthentication> AuthenticationService { get; private set; }
+            internal SteamAuthentication Authentication { get; }
```

Comment:
Can/should this be public?

Reply:
I don't think there is a good reason to expose this.

Reply:
It seems like it could be private/protected then, rather than internal?